Detect devices Huawei, Sony, Irbis, BQ, Lephone, Jay-Tech & detect browser Web Explorer

sanchezzzhak opened the pull request.

sanchezzzhak committed:
Detect: Lephone K7 K10 W7 W9

sanchezzzhak committed:
Detect: Irbis TZ716

sanchezzzhak committed:
Detect: Huawey Enjoy 7 (Nova lite) SLA0AL00, SLA-TL10

sanchezzzhak committed:
Detect: Huawey Mate 10 Pro (BLA-TL00)

sanchezzzhak committed:
Detect: Huawey P20 Pro (CLT-TL00)

sanchezzzhak committed:
Detect: Huawey Honor 8 Pro (DUK-TL30)

sanchezzzhak committed:
Detect: Huawey P20 (EML-TL00)

sanchezzzhak committed:
Detect: Huawey Honor V20 (PCT-TL10)

sanchezzzhak committed:
Detect: Huawey Enjoy 9e (MRD-TL00, MRD-AL00)

sanchezzzhak committed:
Detect: Huawey Honor 7i (ATH-UL00)

sanchezzzhak committed:
Detect: Huawey Honor 7i (ATH-TL00H)

sanchezzzhak committed:
Detect: Etuline devices (ETL-S5042, ETL-S4521, ETL-T752G, ETL-T882G, ETL-S6022, ETL-S3520, ETL-T980)

sanchezzzhak committed:
Detect: Etuline ETL-T970

sanchezzzhak committed:
Detect: Syrox tables SYX-T704, SYX-T101, SYX-T700, SYX-T102

sanchezzzhak committed:
Detect: Kaan devices: N1, N2, A1

sanchezzzhak committed:
Detect: Medion devices: E1041X, E1051X

sanchezzzhak committed:
Detect: Irbis TZ772

sanchezzzhak committed:
Detect: Huawey Honor 4X (CHE2-L12)

sanchezzzhak committed:
Detect: Huawey MediaPad X2 (GEM-703L)

sanchezzzhak committed:
Detect: Huawey MediaPad X2 (GEM-702L)

sanchezzzhak committed:
Detect: Huawei P8 Lite (2015) (ALE-TL00)

sanchezzzhak committed:
Detect: JAY-Tech tables TPCY-TXE*

sanchezzzhak committed:
Detect: BQ devices (BQS-4503, BQS-4550, BQS-3510, BQS-4001,
BQS-4004, BQS-4008, BQS-4009, BQS-4010, BQS-4502, BQS-4516, BQS-5501,
BQS-5002, BQS-4800, BQS-4555, BQS-4525, BQS-4560, BQS-4570, BQS-4707)
BQ models sort regex by number desc

sgiehl commented on regexes/device/mobiles.yml:
```diff
@@ -5832,6 +5891,17 @@ Leotec:
+      device: 'feature phone'
+      model: '$1'
+    - regex: 'lephone[ _](W[79])[/;) ]'
+      model: '$1'
```
Maybe you could also add this one:
`Dalvik/v3.3.127 Compatible (TVM xx; YunOS 3.0; Linux; U; Android 4.4.4 Compatible; lephone T2 Build/KTU84P)`

sanchezzzhak replied:
Added

sanchezzzhak committed:
Detect: BQ devices (BQS-5006, BQS-5009, BQS-5010, BQS-5011, BQru-1057L,
BQS-3503, BQS-4505, BQS-4005, BQS-4501, BQS-4510, BQS-5000, BQS-5001,
BQS-4702, BQS-4515, BQS-5005, BQS-5030, BQS-5502)

sanchezzzhak committed:
Detect: Lephone T2

sanchezzzhak committed:
Detect: Jay-tech (TPCY-TXTE10D, TPCY-TXTE7D)

sanchezzzhak committed:
Detect: Huawei Mate 20 X (EVR-TL00)
Detect: Huawei Enjoy 6 (NCE-AL10, NCE-AL00)
Detect: Huawei MediaPad M5 lite (JDN2-AL00)
Detect: Huawei MediaPad T5 10 (AGS2-L03)
Detect: Huawei Y9 (2019) (JKM-TL00)

sanchezzzhak committed:
Detect: Huawei Y7 Pro (2019) (DUB-AL20)
Detect: Huawei Y7 (2019) (DUB-TL00)
Detect: Huawei Honor 7S (DUA-LX3)

sanchezzzhak committed:
Detect: Sony Xperia 1 (J8170)

sanchezzzhak committed:
Detect: Sony Xperia 10 Plus (I4213)

sanchezzzhak committed:
Detect: Sony Xperia 10 (I4113)

sanchezzzhak committed:
Detect: Sony Xperia 10 Plus (I3223)

sanchezzzhak committed:
Detect: Sony Xperia 10 (I3123, I3113)

sanchezzzhak committed:
Detect: HTC U11 (2PZC5)

sanchezzzhak committed:
Detect browser: Web Explorer (browser is hidden device name)
Fix PR test 2PZC5

sanchezzzhak committed:
Detect Xiaomi Pocophone F1 (POCO F1)

sanchezzzhak committed:
Detect Xiaolajiao LA2-SN, LA2-S, LA2-L

sanchezzzhak committed:
Detect iLA: X, Silk

sanchezzzhak committed:
Detect Kodak: Tablet 7-10, IM5
Detect LG: X2 (LM-X210K)

sanchezzzhak committed:
Detect LG: G7 ThinQ (LM-G710VM)
Detect LG: G7 One (LM-Q910)
Detect LG: X4 (LM-X410S)
Detect LG: K8 (2018) (LM-X212(G))
Detect LG: V35 (LM-V350N)
